### ISSUE TITLE
PHP8.2: Using ${var} in strings is deprecated, use {$var} instead

### ISSUE THREAD

**Issue by jamisonbryant** [cakephp/authorization#228]
The error:
```
deprecated: 8192 :: Using ${var} in strings is deprecated, use {$var} instead on line 93 of /Users/me/Code/www/myapi/vendor/cakephp/authorization/src/Command/PolicyCommand.php
Stack Trace:

include /Users/me/Code/www/myapi/vendor/composer/ClassLoader.php, line 578
Composer\Autoload\ClassLoader::Composer\Autoload\{closure}() /Users/me/Code/www/myapi/vendor/composer/ClassLoader.php, line 578
Composer\Autoload\ClassLoader::loadClass() /Users/me/Code/www/myapi/vendor/composer/ClassLoader.php, line 432
ReflectionClass::__construct() [internal], line ??
Bake\Plugin::findInPath() /Users/me/Code/www/myapi/vendor/cakephp/bake/src/Plugin.php, line 144
Bake\Plugin::discoverCommands() /Users/me/Code/www/myapi/vendor/cakephp/bake/src/Plugin.php, line 97
Bake\Plugin::console() /Users/me/Code/www/myapi/vendor/cakephp/bake/src/Plugin.php, line 73
Cake\Http\BaseApplication::pluginConsole() /Users/me/Code/www/myapi/vendor/cakephp/cakephp/src/Http/BaseApplication.php, line 237
Cake\Console\CommandRunner::run() /Users/me/Code/www/myapi/vendor/cakephp/cakephp/src/Console/CommandRunner.php, line 144
[main] /Users/me/Code/www/myapi/bin/cake.php, line 12
```

The offending code:
```php
// PolicyCommand.php:92
if ($type === 'table') {
    $className = "{$data['namespace']}\Model\\Table\\${name}${suffix}";
} elseif ($type === 'entity') {
    $className = "{$data['namespace']}\Model\\Entity\\${name}";
}
```

Does anyone care if I either make the fix for PHP 8.2 or just switch to `sprintf()`? I get the above dumped to my terminal every time I run bin/cake 😿

**Comment by markstory** [comment]
> Does anyone care if I either make the fix for PHP 8.2 or just switch to sprintf()? I get the above dumped to my terminal every time I run bin/cake 😿

A pull request would be wonderful. PHP 8 style variable interpolation will work in all of the PHP versions we support.

**Comment by jamisonbryant** [comment]
Actually it appears this change was already made by @swiffer 4 weeks ago, I just had an old version, sorry about that.

I will still submit a PR to suppress additional deprecation warnings I uncovered in the test suite.
